src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -210,3 +210,62 @@ worbox.routing.registerRoute(
   })
 );
 ```
+
+## Make standalone requests using a strategy {: #make-requests }
+
+Most developers will use one of Workbox's
+[strategies](/web/tools/workbox/modules/workbox-strategies) as part of a
+[router](/web/tools/workbox/modules/workbox-routing) configuration. This setup makes it easy to
+automatically respond to specific `fetch` events with a response obtained from the strategy.
+
+However, there are situations where making a request using a strategy outside of the standard router
+setup could be useful. For instance, you might be implementing your own routing logic, or you might
+want to create a composite response that contains information from multiple smaller responses,
+stitched together. There are also situations where you'd normally call
+[`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch)
+directly, but you'd like to take advantage of the
+[plugin integration](/web/tools/workbox/guides/using-plugins) offered by a strategy class.
```

Comment:
I'd replace these two introduction paragraphs - we're in the advanced recipes sections, it should be clear that this isn't a "normal" thing for people to be doing.

> There are situations where you may want to use a strategy in your own router set up or with a fetch() request you've made.
>
> Regardless of the use case, you can use the Workbox strategies in a "standalong" fashion via the `makeRequest()` method.